diff --git a/core/modules/system/system.module b/core/modules/system/system.module
index 3c858f9..b6d49c7 100644
--- a/core/modules/system/system.module
+++ b/core/modules/system/system.module
@@ -260,7 +260,19 @@ function system_theme_suggestions_html(array $variables) {
  */
 function system_theme_suggestions_page(array $variables) {
   $path_args = explode('/', trim(\Drupal::service('path.current')->getPath(), '/'));
-  return theme_get_suggestions($path_args, 'page');
+  $suggestions = theme_get_suggestions($path_args, 'page');
+
+  $http_error_suggestions = [
+    'system.401' => 'page__401',
+    'system.403' => 'page__403',
+    'system.404' => 'page__404',
+  ];
+  $route_name = Drupal::routeMatch()->getRouteName();
+  if (isset($http_error_suggestions[$route_name])) {
+    $suggestions[] = $http_error_suggestions[$route_name];
+  }
+
+  return $suggestions;
 }
 
 /**
